From ce6d05cd8db0f308669de17459b6f474f19806a5 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Mon, 30 Mar 2020 21:14:24 +0800
Subject: [PATCH 05/22] driver: media: imx8: initialize variable mf

It is need to initialize variable mf before using the member
Of this structural variable

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/media/platform/imx8/mxc-isi-cap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/imx8/mxc-isi-cap.c b/drivers/media/platform/imx8/mxc-isi-cap.c
index 5ef49686ae56..05c3cce197ed 100644
--- a/drivers/media/platform/imx8/mxc-isi-cap.c
+++ b/drivers/media/platform/imx8/mxc-isi-cap.c
@@ -1439,7 +1439,7 @@ static int mxc_isi_subdev_get_fmt(struct v4l2_subdev *sd,
 {
 	struct mxc_isi_dev *mxc_isi = v4l2_get_subdevdata(sd);
 	struct mxc_isi_frame *f;
-	struct v4l2_mbus_framefmt *mf;
+	struct v4l2_mbus_framefmt *mf = &fmt->format;
 
 	mutex_lock(&mxc_isi->lock);
 
-- 
2.17.1

